bin/s2s-token.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eu
+
+microservice=${1}
+oneTimePassword=${2}
+
+curl --insecure --fail --show-error --silent -X POST \
+  ${SERVICE_AUTH_PROVIDER_API_BASE_URL:-http://localhost:4502}/lease \
```

Comment:
Rather than using the docker container to generate the OTP, you can just call 
```
token=$(curl $CURL_OPTS -X POST \
  -H "Content-Type: application/json" \
  -d '{"microservice":"'${MICROSERVICE}'"}' \
  http://rpe-service-auth-provider-aat.service.core-compute-aat.internal/testing-support/lease)
``` 
to get a token

Reply:
sure will remove docker and use testing support

Reply:
done